addon/templates/components/ember-power-select/multiple.hbs

```diff
@@ -8,7 +8,11 @@
     {{#if resultsLength}}
       {{#ember-power-select/options options=results highlighted=_highlighted selection=selection
         highlight=(action "highlight") select=(action "select") dropdown=dropdown as |option|}}
-        {{yield option}}
+        {{#if baseHasBlock}}
+          {{yield option}}
+        {{else}}
+          {{if optionLabelPath (get option optionLabelPath) option}}
```

Comment:
can we save that if? At the end the user has to provide either a block or  the optionLabelPath. If the user provides none an assertion should warn him.